imageregistry/v1/types.go

```diff
@@ -303,8 +302,60 @@ type ImageRegistryConfigStorageAzure struct {
 	// object.
 	// +optional
 	CloudName string `json:"cloudName,omitempty"`
+	// networkAccess defines the network access properties for the storage account.
+	// type: Internal means the storage account will be private, type: External
```

Comment:
The lines here and over Internal mention duplicit information about type internal and how the underlying fields are set. Up to you but it's better to avoid duplicities unless they give additional details that are not mentioned by default where the data type is defined.

Reply:
moved to the `Type` definition over at `AzureNetworkAccess`.